README-trust.md

```diff
@@ -0,0 +1,118 @@
+Trust module
+============
+
+Description
+-----------
+
+The trust module allows to ensure presence and absence of a domain trust.
+
+Features
+--------
+
+* Trust management
+
+Supported FreeIPA Versions
+--------------------------
+
+FreeIPA versions 4.4.0 and up are supported by the ipagroup module.
+
+Requirements
+------------
+
+**Controller**
+
+* Ansible version: 2.8+
+
+**Node**
+
+* Supported FreeIPA version (see above)
+* samba-4
+* server-trust-ad
+
+Usage
+=====
+
+Example inventory file
+
+```ini
+[ipaserver]
+ipaserver.test.local
+```
+
+Example playbook to create a one-way trust:
+
+```yaml
+---
+- name: Playbook to create a trust
+  hosts: ipaserver
+  become: true
+
+  tasks:
+  - name: create a one-way trust to the windows.local domain
+    ipatrust:
+      realm: windows.local
+      admin: Administrator
+      password: secret_password
+      state: present
+```
+
+Example playbook to create a two-way trust using a shared-secret:
+
+```yaml
+---
+- name: Playbook to create a trust
+  hosts: ipaserver
+  become: true
+
+  tasks:
+  - name: create a two-way trust to the windows.local domain
+    ipatrust:
+      realm: windows.local
+      trust_secret: my_share_Secret
+      two_way: True
+      state: present
+```
+
+Example playbook to delete a trust:
+
+```yaml
+---
+- name: Playbook to delete trust
+  hosts: ipaserver
+  become: true
+
+  tasks:
+  - name: delete the trust with windows.local
+    ipatrust:
+      realm: windows.local
+      state: absent
+```
+
+This will only delete the ipa-side of the trust and it does NOT delete the id-range that matches the trust,  
+
+Variables
+=========
+
+ipatrust
+-------
+
+Variable | Description | Required
+-------- | ----------- | --------
+`ipaadmin_principal` | The admin principal is a string and defaults to `admin` | no
+`ipaadmin_password` | The admin password is a string and is required if there is no admin ticket available on the node | no
+`realm` | The realm name string. | yes
+`admin` | Active Directory domain administrator string. | no
+`passwordd` | Active Directory domain administrator's password string. | no
```

Comment:
Is the second `d` on password a typo? It should be "password".

Reply:
fixed